tests/aws/services/transcribe/test_transcribe.py

```diff
@@ -403,3 +405,39 @@ def test_transcribe_start_job_same_name(
             TranscriptionJobName=transcribe_job_name
         )
         snapshot.match("delete-transcription-job", res_delete_transcription_job)
+
+    @pytest.mark.parametrize(
+        "media_file, max_speakers",
+        [
+            ("../../files/multi-speaker.wav", 2),
+            ("../../files/multi-speaker.wav", 1),
+        ],
+    )
+    @markers.aws.validated
+    @pytest.mark.skipif(condition=not is_aws_cloud(), reason="Not yet implemented on LS")
+    def test_transcribe_speaker_diarization(
+        self, transcribe_create_job, media_file, max_speakers, aws_client, snapshot
+    ):
+        file_path = os.path.join(BASEDIR, media_file)
+        settings = {"Settings": {"MaxSpeakerLabels": max_speakers, "ShowSpeakerLabels": True}}
+
+        if max_speakers < 2:
+            with pytest.raises(ParamValidationError) as e:
+                transcribe_create_job(audio_file=file_path, params=settings)
+            snapshot.match("err_speaker_labels_diarization", e.value)
+            return
```

Comment:
It's a good practice to make a test only test a certain behaviour. It would be cleaner to assert this exception in a separate test.